selenium_tests/course_admin/page_objects/course_admin_dashboard_page_object.py

```diff
@@ -6,6 +6,7 @@
 
 
 class Locators(object):
+    CLASS_ROSTER_BUTTON = (By.ID, "course-roster")
```

Comment:
(Darn it! :) We forgot to change the ID from `course-roster` to `class-roster`!)